ui/src/containers/DashboardPage.js

```diff
@@ -122,11 +113,15 @@ const DashboardPage = (props: {}) => {
     onClick: () => {
       writeUrlTimeSpan(option);
     },
-    selected: queryTimeSpansCodes.find(timespan => timespan.duration === metricsTimeSpan)?.label === option,
+    selected:
+      queryTimeSpansCodes.find((item) => item.label === label)?.label ===
```

Comment:
Just FYI:
I exposed the `queryTimeSpansCodes` from `core-ui` as well ;) To make sure we have a single source of truth.